DirectXTex/DirectXTexMipmaps.cpp

```diff
@@ -2521,6 +2521,129 @@ namespace
 // Entry-points
 //=====================================================================================
 
+float clamp(float value, float min, float max)
```

Comment:
These new helper functions should be higher up (not the entry-points section) and in an anonymous namespace